Automattic/jetpack
# Search: Handle pagination edge case with excluded post types

#### Changes proposed in this Pull Request:
* Overrides incorrect API response value for paginated queries with excluded post types. This can occur when paginating with excluded post types configured via the Customizer. 

#### Jetpack product discussion
p8oabR-DS-p2#comment-5144

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* If possible, override the site search ID (via mu-config.php) to `191431359`.
* Ensure that you have WooCommerce installed on your site.
* Through the customizer, exclude posts, pages, and attachments (media).
* Summon the search modal and set the search order to `oldest`; use an empty search query (`/?s=&sort=oldest`).
* Scroll to the bottom of the modal to load more results. Repeat this until all results have loaded. (Without this bugfix, the results will disappear following the final pagination API request.)
